Makefile

```diff
@@ -78,29 +114,43 @@ prepare-version: version.txt
 
 version.txt:
 	@echo "2.$$(git rev-list HEAD --count).$$(git rev-parse --short HEAD | perl -ne 'print hex $$_')" > $@
+	@# Replace version in pyproject.toml
+	sed -ie "s/__version__ = \"[0-9\.]\+\"/__version__ = \"`cat $@`\"/" src/chtools/__init__.py
 
+.PHONY: build-deb-package
 build-deb-package: prepare-changelog
 	cd debian && debuild --check-dirname-level 0 --preserve-env --no-lintian --no-tgz-check -uc -us
 
+.PHONY: clean
+clean:
+	@echo 'Cleaning up'
+
+	rm -rf build
+	rm -rf debian/files debian/.debhelper
+	rm -rf debian/mdb-ch-tools*
+
 .PHONY: help
 help:
 	@echo "Base targets:"
 	@echo "  prepare-changelog          Add an autobuild version entity to changelog"
 	@echo "  prepare-version            Update version based on latest commit"
+	@echo "  build-python-package       Build 'ch-tools' Python package"
 	@echo "  build-deb-package          Build 'mdb-ch-tools' debian package"
+	@echo "  clean                      Clean up after building debian package"
 	@echo ""
 	@echo "--------------------------------------------------------------------------------"
 	@echo ""
 	@echo "Debian package build targets:"
 	@echo "  install                    Install 'mdb-ch-tools' debian package"
 	@echo "  uninstall                  Uninstall 'mdb-ch-tools' debian package"
 	@echo ""
+	@echo "  install-python-package     Install 'ch-tools' python package"
 	@echo "  uninstall-python-package   Uninstall 'ch-tools' python package"
 	@echo "  install-symlinks           Install symlinks to /usr/bin/"
 	@echo "  uninstall-symlinks         Uninstall symlinks from /usr/bin/"
```

Comment:
Btw `flit install` also handles symlinking to executables, so you wouldn't need a separate step for that

Reply:
These executables are installed on host's `/usr/bin`, not in `$DESTDIR/usr/bin`